<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormDicom</class>
 <widget class="QWidget" name="FormDicom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>206</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dicominfo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/basicviewer.qrc">
    <normaloff>:/images/main.ico</normaloff>:/images/main.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: #F4D3A5;
	color: #FFFFFF;
}

QSlider::handle::vertical {
	height: 16px;
	background: #FFFFFF;
	border: #000066;
	border-radius: 0px;
}

QSlider::groove::vertical {
	background-color: #353662;
	width: 8px;
	border: 1px solid #7374A0;
}

QSlider::handle::horizontal {
	width: 16px;
	background: #FFFFFF;
	border: #000066;
	border-radius: 0px;
}

QSlider::groove::horizontal {
	background-color: #353662;
	height: 8px;
	border: 1px solid #7374A0;
}

QPushButton {
	border: 1px solid #8f8f91;
	color: #444444;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #f6f7fa, stop: 1 #dadbde);
	padding-bottom: 2px;
	margin: 1px;
	padding: 2px;
	font-size: 14px;
}

QPushButton:pressed {
	color: #000000;
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #dadbde, stop: 1 #f6f7fa);
}

QFrame {
	background-color: #FFFFFF;
	color: #000000;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>21</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: #FAAD5C;
}

QLabel {
	font: 75 10pt &quot;Arial&quot;;
	font-weight: bold;
	color: #140865;
}

QToolButton { 
	border: 0px;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lblTitle">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
        <property name="indent">
         <number>9</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnClose">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../qrc/basicviewer.qrc">
          <normaloff>:/images/close_dialog.png</normaloff>:/images/close_dialog.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="margin">
      <number>9</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="txtDicominfo">
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc/basicviewer.qrc"/>
 </resources>
 <connections/>
</ui>
